edward/util/tensorflow.py

```diff
@@ -310,6 +310,44 @@ def multivariate_rbf(x, y=0.0, sigma=1.0, l=1.0):
       tf.exp(-1.0 / (2.0 * tf.pow(l, 2.0)) * tf.reduce_sum(tf.pow(x - y, 2.0)))
 
 
+def multivariate_rbf_kernel(x, sigma=1.0, l=1.0):
```

Comment:
multivariate_rbf_kernel() might be needed for any model with a RBF kernel such as GP, Cox. 

in the long term i think we might want to implement other kernels and not just RBF. Probably not in util but in a kernel.py file as is done in GPflow. Then in the example scripts we can simply do:

from ed.kernel import rbf, linear
K = rbf(x)
K_lin = linear(x)